tutorials/fluid_modeling.ipynb

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
<p>Perhaps move some of this towards the top, to gather such information when natural. Also 'overall fraction' must be defined somewhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I have introduced links to the respective quantities implemented in the code. The docstring contain sufficient information to make it clear what it is.</p><p>Is that fine?</p><p> </p><p>Since this tutorial is a bit extensive, I am trying to not burden it further with formulas and definitions, but focus on the application of the code and link to the respective parts. The documentation of the compositional framework is very extensive, I want to make use of it.</p>

Reply:
<p>I agree with the overall idea, we are not Wikipedia. Since the links just bring me to the top of the file, perhaps it is better to give this as an instruction, such as 'an explanation can be found in the documentation, see here'. </p>

Reply:
The link should lead directly to the respective line. Have you tried that in vs code or online?
On my machine, locally in VS code, it leads to the correct line.